xuluomin/src/client/src/client.cpp

```diff
@@ -0,0 +1,73 @@
+
+#include "ros/ros.h"
+#include "std_msgs/String.h"
+#include <ros/time.h>
+#include <client/time.h>
+#include <client/show.h>
+#include <signal.h>
+#include <unistd.h>
+#include <iostream>
+void interrupt_handler(int x);
+bool flag_ros = 1;
+ros::ServiceClient service_client;
+client::show srv;
+int main(int argc, char **argv){
+    std::string name;
+    //input the name
+    std::cin>>name;
+    // 初始化ROS节点
+    ros::init(argc, argv, name);
+    // 创建节点句柄
+    ros::NodeHandle n;
+
+    // 初始化client::show的请求数据
+    srv.request.request = 1;
+    srv.request.name = name;
+    srv.response.response = 0;
+    
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]